providers/os/connection/device/linux/device_manager.go

```diff
@@ -443,7 +474,12 @@ func (c *LinuxDeviceManager) identifyDeviceViaLun(lun int) ([]snapshot.BlockDevi
 }
 
 func (c *LinuxDeviceManager) identifyViaDeviceName(deviceName string, mountAll bool, includeMounted bool) ([]*snapshot.PartitionInfo, error) {
-	blockDevices, err := c.volumeMounter.CmdRunner.GetBlockDevices()
+	if deviceName == "" {
```

Comment:
what's the use case for no device name?

Reply:
Just a failsafe, i've hit this case when used --device-name flag wrong and that just adds false errors, where it shouldn't